e2e_test/extended_mode/type.slt

```diff
@@ -3,10 +3,10 @@
 statement ok
 SET RW_IMPLICIT_FLUSH TO true;
 
-# RisingWave can't support list and struct now so we skip them.
-# include ../batch/types/array.slt.part
-# include ../batch/types/struct.slt.part
-# include ../batch/types/list.slt.part
```

Comment:
Seems like we removed some commented lines:
```
# include ../batch/types/list.slt.part
# include ../batch/types/array.slt.part
```

If these are no longer required we should remove these test files as well.